src/view/com/lightbox/ImageViewing/components/ImageItem/ImageItem.ios.tsx

```diff
@@ -128,50 +110,45 @@ const ImageItem = ({
       runOnJS(handleDoubleTap)(absoluteX, absoluteY)
     })
 
-  const composedGesture = Gesture.Exclusive(doubleTap, singleTap)
+  const composedGesture = Gesture.Exclusive(
+    dismissSwipePan ?? Gesture.Manual(),
+    doubleTap,
+    singleTap,
+  )
 
   return (
     <GestureDetector gesture={composedGesture}>
       <Animated.ScrollView
         // @ts-ignore Something's up with the types here
         ref={scrollViewRef}
-        style={styles.listItem}
         pinchGestureEnabled
         showsHorizontalScrollIndicator={false}
         showsVerticalScrollIndicator={false}
         maximumZoomScale={maxZoomScale}
-        onScroll={scrollHandler}>
-        <Animated.View style={[styles.imageScrollContainer, animatedStyle]}>
-          <ActivityIndicator size="small" color="#FFF" style={styles.loading} />
-          <Image
-            contentFit="contain"
-            source={{uri: imageSrc.uri}}
-            placeholderContentFit="contain"
-            placeholder={{uri: imageSrc.thumbUri}}
-            style={styles.image}
-            accessibilityLabel={imageSrc.alt}
-            accessibilityHint=""
-            enableLiveTextInteraction={showControls && !scaled}
-            accessibilityIgnoresInvertColors
-          />
-        </Animated.View>
+        onScroll={scrollHandler}
+        bounces={scaled}
+        centerContent>
+        <ActivityIndicator size="small" color="#FFF" style={styles.loading} />
+        <Image
+          contentFit="contain"
+          source={{uri: imageSrc.uri}}
+          placeholderContentFit="contain"
+          placeholder={{uri: imageSrc.thumbUri}}
+          style={{
+            width: SCREEN.width,
+            height: imageAspect ? SCREEN.width / imageAspect : undefined,
```

Comment:
new layout stuff. i needed to figure out how to position the scroll view so that its scrolling container doesn't stretch all the way to the top and bottom of the screen. this is important because it fixes an old iOS issue. previously, it was possible to zoom in and then vertically move past the black bars. so it was hard to zoom into corners. now this is fixed.

the price for that is that we won't show the image _until_ we've loaded its dimensions (because we don't know the height). this is generally ok now because https://github.com/bluesky-social/social-app/pull/6073 plumbs dimensions from `aspectRatio` in the majority of cases. (in other cases, we'll wait for the `Image.getSize` to call back — which means there'll be no nice transition and you'll see a spinner — but it's no worse than status quo.)